[
    {
        "content": "<p>I'm getting the following error from the IG publisher at the end of the build:</p>\n<div class=\"codehilite\"><pre><span></span><code>Internal error in location for message: &#39;Error in ?? at 1, 1: Found / expecting a token name&#39;, loc = &#39;/Users/elliot/src/covid-ig/output/Questionnaire-WhoCrQuestionnaireCovid19Surveillance.html#/html/bod&#39;, err = &#39;The image source &#39;icon-q-dateTime.png&#39; cannot be resolved&#39;\n</code></pre></div>\n<p>Looking in the output directory, there is an <code>icon-q-datetime.png</code>, but no <code>icon-q-dateTime.png</code>.</p>",
        "id": 215981194,
        "sender_full_name": "Elliot Silver",
        "timestamp": 1604796069
    },
    {
        "content": "<p><span class=\"user-mention\" data-user-id=\"191316\">@Grahame Grieve</span>, would this be a publisher issue or a template issue?</p>",
        "id": 216427084,
        "sender_full_name": "Elliot Silver",
        "timestamp": 1605155832
    },
    {
        "content": "<p>publisher, I thik</p>",
        "id": 216429884,
        "sender_full_name": "Grahame Grieve",
        "timestamp": 1605159388
    },
    {
        "content": "<p>It's on my list.</p>",
        "id": 216429886,
        "sender_full_name": "Grahame Grieve",
        "timestamp": 1605159391
    },
    {
        "content": "<p>Thank you.</p>",
        "id": 216429983,
        "sender_full_name": "Elliot Silver",
        "timestamp": 1605159490
    },
    {
        "content": "<p>From what I can tell, the filename in the pubpack has different case than what the publisher (jekyll?) is looking for.</p>",
        "id": 216430484,
        "sender_full_name": "Elliot Silver",
        "timestamp": 1605160161
    },
    {
        "content": "<p>ok I think the next release will fix this</p>",
        "id": 216561383,
        "sender_full_name": "Grahame Grieve",
        "timestamp": 1605236384
    }
]